net_gen/src/lib.rs

```diff
@@ -2,7 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the THIRD-PARTY file.
 
-#![allow(clippy::all)]
```

Comment:
The wording of the commit message would be better to say something like "net_get: Use more specific clippy lint controls" There was nothing incorrect about the previous one; just too broad.